catalog/ui/catalog-ui-search/webpack/config/base.js

```diff
@@ -51,6 +64,11 @@ module.exports = {
         }
     },
     plugins: [
+        new webpack.DefinePlugin({
```

Comment:
Should we communicate that this information is only accurate as of starting the dev server? As soon as any files are modified, this information is potentially out of date.

Reply:
Yeah, I was more interested in helping out actual builds with the commit hash / date information.  I could put that disclaimer in the README though.